charts/studio/templates/deployment-studio-ui.yaml

```
@@ -45,7 +45,11 @@ spec:
             periodSeconds: 10
           readinessProbe:
             httpGet:
-              path: /api/health
+            {{- if and .Values.global.basePath (not (eq .Values.global.basePath "/")) }}
+              path: /{{ include "studio.basePath" . }}/fapi/health
+            {{- else }}
+              path: /fapi/health
+            {{- end }}
```

```suggestion
            {{- if and .Values.global.basePath (not (eq .Values.global.basePath "/")) }}
              path: /{{ include "studio.basePath" . }}/{{- if semverCompare ">=1.80.0" (.Values.studioUi.image.tag | default .Chart.AppVersion) }}f{{end}}api/health
            {{- else }}
              path: /{{- if semverCompare ">=1.80.0" (.Values.studioUi.image.tag | default .Chart.AppVersion) }}f{{end}}api/health
            {{- end }}
```

It will check if the semVer of the app or UI tag is `< 1.80.` then will use the `/api/health` 
If  `>= 1.80.0` will use `fapi`. 

The 1.80.0 version (or any other) will be a version, where PathBase will be published

Reference:
https://oscarchou.com/posts/howto/helm-chart-use-semantic-version/
https://helm.sh/docs/chart_template_guide/function_list/